### PR TITLE
Read IPFS port mappings from env vars

### PR DESCRIPTION
**Pull request by FabijanC**
So far the ports were hardcoded. How should the variables be named?

### CODE REVIEW

File: environments/.env.latest [hunk 1]
```diff
@@ -39,7 +39,10 @@ BUCKET_NAME=s3://sourcify-staging
 
 # IPFS config
 IPFS_SECRET=xxx
-IPFS_EXTERNAL_PORT=5050
+IPFS_EXTERNAL_PORT_8080=5050
+IPFS_EXTERNAL_PORT_4001=4001
+IPFS_EXTERNAL_PORT_4002_UDP=4002
+IPFS_EXTERNAL_PORT_5001=5001
```
Comment (Jakic007) [inline environments/.env.latest:45]: IPFS_API_EXTERNAL_PORT

File: environments/.env.latest [hunk 2]
```diff
@@ -39,7 +39,10 @@ BUCKET_NAME=s3://sourcify-staging
 
 # IPFS config
 IPFS_SECRET=xxx
-IPFS_EXTERNAL_PORT=5050
+IPFS_EXTERNAL_PORT_8080=5050
+IPFS_EXTERNAL_PORT_4001=4001
```
Comment (Jakic007) [inline environments/.env.latest:43]: IPFS_LIBP2P_EXTERNAL_PORT

File: environments/.env.latest [hunk 3]
```diff
@@ -39,7 +39,10 @@ BUCKET_NAME=s3://sourcify-staging
 
 # IPFS config
 IPFS_SECRET=xxx
-IPFS_EXTERNAL_PORT=5050
+IPFS_EXTERNAL_PORT_8080=5050
```
Comment (Jakic007) [inline environments/.env.latest:42]: IPFS_GW_EXTERNAL_PORT

File: environments/.env.latest [hunk 4]
```diff
@@ -39,7 +39,10 @@ BUCKET_NAME=s3://sourcify-staging
...
+IPFS_EXTERNAL_PORT_4001=4001
+IPFS_EXTERNAL_PORT_4002_UDP=4002
```
Comment (Jakic007) [inline environments/.env.latest:44]: This one has no purpose IMO.

Comment (FabijanC) [reply environments/.env.latest:44]: Then I'll remove all it from the .env files and the mapping from ipfs.yaml.

**Pull request merged**